projects/addon-commerce/components/input-card-grouped/input-card-grouped.component.ts

```diff
@@ -218,7 +218,10 @@ export class TuiInputCardGroupedComponent
     }
 
     get icon(): PolymorpheusContent {
-        return this.cardSrc || this.defaultIcon;
+        return (
+            (this.cardSrc as Exclude<PolymorpheusContent, null | undefined>) ||
```

Comment:
Sometimes we want empty string to be considered falsy values and keep `||` rather than `??`. Maybe we should move this rule to warning?